auto_submit/test/requests/github_webhook_test_data.dart

```diff
@@ -87,3 +498,386 @@ String reviewsMock = '''[
     "author_association": "OWNER"
   }
 ]''';
+
+String checkRunsMock = '''{
```

Comment:
Hi Casey, here we need checkRuns mock and checkSuites mock because we need to query to get the check runs and check suites in the github_service. Therefore, in the fake_github_service, when we override the getCheckRuns and getCheckSuites methods, we need to give it a return data, which should be the data from these two mocks data.

Reply:
It would be easier if you create the Dart objects directly than import huge chunks of json. This code is brittle and will easily break with changes to the API, and it will unfortunately fail silently making it hard to debug.

Reply:
Hi Casey, yes I think creating the Dart objects directly is a good idea here. I will replace the checkRunsMock to List<checkRuns> in fake_github_service to avoid the influence of changing API.

Reply:
Hi Casey, when I tried to create the CheckRun object directly instead of using a chunk of json, I realized that the only constructor I can use from CheckRun class is CheckRun.fromJson() which still needs the json data. Therefore, I will keep the Json data here.

Reply:
Sgtm. Can you file an issue and I can upstream a constructor so we make it easier

Reply:
Sure, here is the issue:https://github.com/flutter/flutter/issues/99717. Thank you Casey!